modules/media/ajax/FileUpload.php

```diff
@@ -107,9 +107,14 @@ function uploadFile()
         showError("Please fill in all required fields!");
         return;
     }
+    $fileName  = str_replace(' ', '_', trim($_FILES["file"]["name"]));
```

Comment:
Good call, I first trimmed whitespace here, but then realized it needs to be done on the front-end too. So I guess I'll just remove it from here to avoid disparity in the future